File: FindtheTownJudge.cpp
class Solution {
public:
    int findJudge(int N, vector<vector<int>>& trust) {
        
        vector<int>freq(N+1,0);
        
        for(auto t:trust)
        {
            freq[t[0]]--;
            freq[t[1]]++;
            
        }
        for(int i=1;i<=N;i++)
        {
            if(freq[i]==N-1)
                return i;
        }
        return -1;
    }
};

File: Find the Town Judge.py
# Day : 10

class Solution:
    def findJudge(self, N: int, trust: List[List[int]]) -> int:
        trusts = [0] * (N+1)
        for (a, b) in trust:
            trusts[a] -= 1
            trusts[b] += 1
            
        for i in range(1, len(trusts)):
            if trusts[i] == N-1:
                return i
        return -1
